.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,160 @@
+name: benchmarks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled # run-benchmarks label required
```

Comment:
why not just `benchmark`?

Reply:
Done in https://github.com/tardis-sn/tardis/pull/2260/commits/df6404668666c79a422b58be72aadd19e747fd38